app/controllers/users_controller.rb

```diff
@@ -6,9 +6,26 @@ class UsersController < ApplicationController
   before_action :authenticate_user!, except: %i(new create public user_photo)
   respond_to :html
 
+  # The publisher/aspect_dropdown partial requiers a @stream that responds to publisher().public()
```

Comment:
I agree that it would be much nicer if the dropdown does not depend on the presence of a @stream. I wanted to keep the impact of this change limited. It seems that more refactoring is required as a prerequisite for this change, otherwise the aspect_dropdown can not be used anywhere but inside a publisher in a stream.